lambda_layers/core_layer/python/core_layer/db_handler.py

```diff
@@ -35,6 +35,14 @@ def creator(): return sqlite3.connect(
 Session = scoped_session(session_factory)
 
 
+def substring(column, delimeter, session):
+    if session.bind.dialect.name == 'sqlite':
+        return func.iif(func.instr(column, delimeter) > 0, func.substr(
+            column, 1, func.instr(column, delimeter)-1), column)
+    elif session.bind.dialect.name == 'mysql+auroradataapi':
+        return func.substring_index(column, delimeter, 1)
```

Comment:
Hätte die Funktion spontan in einer anderen Datei erwartet - irgendwas wie `db_helper` o.Ä.

Reply:
> Ist das nicht potentiell gefährlich? Es kann doch durchaus sein, dass eine Seite Informationen zu einem Artikel im Query Parameter hat, oder?

Ja, schon. Die alternativen sind (Stand jetzt) aber folgende.
1. Nur genauer Abgleich der URL (wie bisher)--> Wenn Artikel mit queryParams eingereicht werden, wird der geschlossene Artikel nicht gefunden
2. Abgleich mit "contains" --> Dann würde bspw. "www.zeit.de" alle Artikel der Domain triggern. 

Habe mit Max besprochen, dass es so wahrscheinlich vorerst der beste Weg ist - Fall 1 gibt es nämlich auf jeden Fall schon in unseren Daten.

Reply:
> Hätte die Funktion spontan in einer anderen Datei erwartet - irgendwas wie `db_helper` o.Ä.

Hab ich erstellt